lib/cli/cliUpdater.js

```diff
@@ -68,10 +76,43 @@ class CliUpdater {
     const currentVersionParts = currentVersion.split(".").map(Number);
 
     for (let i = 0; i < latestVersionParts.length; ++i) {
-      // Compare parts of the version numbers and return the result if they are different
       if (latestVersionParts[i] !== currentVersionParts[i]) {
-        return latestVersionParts[i] > currentVersionParts[i] ? true : false;
+        return latestVersionParts[i] > currentVersionParts[i];
+      }
+    }
+  }
+
+  static async #getRecentChanges() {
+    try {
+      const response = await axios.get(CHANGELOG_URL);
+      if (response.status !== 200) {
+        return null;
       }
+
+      const changelog = response.data;
+
+      // Split the changelog into version sections, which are separated by a line with "## ["
+      const versionSections = changelog.split(/## \[/);
+
+      // Get the last 3 version sections
+      const recentVersions = versionSections.slice(1, 4);
+
+      // Format the changes for the output
+      let changes = "";
+      for (const version of recentVersions) {
+        const versionNumber = version.split(")")[0];
+        changes += chalk.bold(`\n[${versionNumber})\n`);
+
+        // Get the content of the version section
+        const changeContent = version.split(")")[1];
+        if (changeContent) {
+          changes += changeContent.trim() + "\n";
+        }
+      }
+
+      return changes;
+    } catch (err) {
+      return null;
     }
```

Comment:
Some personal opinions here:

I think this could be a good candidate to have a separate service class. Something that we can call as, for example,  `ChangelogReader.fetchChanges(previousVersion, currentVersion)` here. (names of the class or method can be different :smile: )

Considering that we know current and previous version I would try to display entries between them instead of a fix amount of entries.

We should also add tests now that we are introducing it.